corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/navigation.scss

```diff
@@ -12,6 +12,7 @@
 
 #hq-navigation {
   transition: all 1s;
+  z-index: 1060;
```

Comment:
I think this can be avoided if we kept the z-index of persistent menu under 1000. Is there a particular reason you made persistent menu z-index >1000?

Reply:
After some digging, I believe Addison got to that value because some other elements could be 1000+, for instance  `query-button-container` has a z-index of 1020. When the persistent men nav isn't locked it's supposed to expand over whatever is on the page.

Reply:
A deep dive into existing z-index vars might be a good individual initiative ticket. Things could probably be cleaned up and reorganized.

Reply:
I see, just slightly concerned about any other unknown effects of adding z-index to `hq-navigation` since that's used in all of HQ. But not that concerned that it's a blocker.

It would be good to add this z-index into the list of z-indexes. Just to make clean up easier whenever that is done.